Rakefile

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env rake
+
+desc "Update vendor/cookbooks"
+task :vendor do
+  puts "Vendoring cookbooks"
+  `rm -rf vendor`
+  `berks update`
+  `berks vendor vendor/cookbooks`
+end
+
+desc "Package cookbooks into a chef-solo tarball"
+task :package => :vendor do
+  `mkdir -p vendor/cookbooks/conjur`
+  `cp -r metadata.rb Berksfile Berksfile.lock CHANGELOG.md chefignore README.md attributes recipes spec vendor/cookbooks/conjur`
+  version=`git describe --tags --abbrev=0`.strip
```

Comment:
I don't think we want `--abbrev` here. This will make the version release not always match the tag.